server/views/patterns/typography.njk

```diff
@@ -1,27 +1,379 @@
+  <div class="example">
+    <ul>
+      <li>
+        United Kingdom of Great Britain and Northern Ireland:
+        <ul>
+          <li>England</li>
+          <li>Scotland</li>
+          <li>Wales</li>
+          <li>Northern Ireland</li>
+        </ul>
+      </li>
+      <li>Republic of Ireland</li>
+      <li>Isle of Man</li>
+      <li>
+        Channel Islands:
+        <ul>
+          <li>Bailiwick of Guernsey</li>
+          <li>Bailiwick of Jersey</li>
+        </ul>
+      </li>
+    </ul>
+  </div>
+  <p>Sometimes we may want each list item to contain block elements, typically a paragraph or two.</p>
+  <div class="example">
+    <ul>
+      <li>
+        <p>The British Isles is an archipelago consisting of the two large islands of Great Britain and Ireland, and many smaller surrounding islands.</p>
+      </li>
+      <li>
+        <p>Great Britain is the largest island of the archipelago. Ireland is the second largest island of the archipelago and lies directly to the west of Great Britain.</p>
+      </li>
+      <li>
+        <p>The full list of islands in the British Isles includes over 1,000 islands, of which 51 have an area larger than 20 km<sup>2</sup>.</p>
+      </li>
+    </ul>
+  </div>
+
+  <h3>Definition list</h3>
+  <p>The <code>dl</code> element is for another type of list called a definition list. Instead of list items, the content of a <code>dl</code> consists of <code>dt</code> (Definition Term) and <code>dd</code> (Definition description) pairs. Though it may be called a &#8220;definition list&#8221;, <code>dl</code> can apply to other scenarios where a parent/child relationship is applicable. For example, it may be used for marking up dialogues, with each <code>dt</code> naming a speaker, and each <code>dd</code> containing his or her words.</p>
+  <div class="example">
+    <dl>
+      <dt>This is a term.</dt>
+      <dd>This is the definition of that term, which both live in a <code>dl</code>.</dd>
+      <dt>Here is another term.</dt>
+      <dd>And it gets a definition too, which is this line.</dd>
+      <dt>Here is term that shares a definition with the term below.</dt>
+      <dt>Here is a defined term.</dt>
+      <dd><code>dt</code> terms may stand on their own without an accompanying <code>dd</code>, but in that case they <em>share</em> descriptions with the next available <code>dt</code>. You may not have a <code>dd</code> without a parent <code>dt</code>.</dd>
+    </dl>
+  </div>
+
+  <h3>Figures</h3>
+  <p>Figures are usually used to refer to images:</p>
+  <div class="example">
+    <figure>
+      <img src="http://lorempixel.com/680/400/abstract" alt="Example image" />
```

Comment:
Is there a reason we don't start from nothing and build up?
Not fussed, just wondering.